src/SFA.DAS.Employer.PR.Domain/Interfaces/IOuterApiClient.cs

```diff
@@ -14,4 +14,6 @@ public interface IOuterApiClient
     [Get("/relationships/employeraccount/{accountHashedId}")]
     Task<GetEmployerRelationshipsQueryResponse> GetAccountLegalEntities([Path] string accountHashedId, CancellationToken cancellationToken);
 
+    [Get("/providers")]
+    Task<GetProvidersResponse> GetProviders(CancellationToken cancellationToken);
```

Comment:
Rename suggestion `GetRegisteredProviders`, if you choose rename, also consider Response class

Reply:
updated